main/acle.rst

```diff
@@ -5792,3 +5811,39 @@ Instructions
 +---------------------------------+----------------+------------+-----------------+
 
 These intrinsics are available when ``arm_acle.h`` is included.
+
+.. _ssec-MOPS-intrinsics:
+
+memcpy family of memory operations (MOPS) intrinsics
+####################################################
+
+Introduction
+============
+
+This section describes the intrinsics for the new instructions introduced
+for memcpy, memmove and memset family of memory operations (MOPS).
+
+These intructions were designed to enable the standarisation of the software
+implementation of those operations. Therefore, most of the use cases for
+the new instructions should be covered by the compiler's code generation or
+by library implementations.
+
+An exception to that is the set of instructions covering the memset operation
+with memory tagging. An intrinsic is available to provide access to this
+operation. See ssec-MTE_ for more information on memory tagging.
+
+The ``<arm_acle.h>`` header should be included before using this intrinsic.
+
+Intrinsics
+==========
+
+This intrinsic is available when ``__ARM_FEATURE_MOPS`` is defined.
+::
```

Comment:
This should use the `.. code: c` highlighting. Also, please leave a blank line between after the text.

Reply:
I've updated it to use the `.. code:: c` directive, but I've noticed the other snippets of the spec don't make use of it. Should this one follow the direction of the others for consistency?

Reply:
Oh yes, please, do adhere to the current way code snippets are produced in this doc. Sorry about that!

Reply:
No worries, I've reverted it to match the other ones.